homeassistant/components/thermostat/vera.py

```diff
@@ -0,0 +1,86 @@
+"""
+Support for Vera thermostats.
```

Comment:
Please don't add it as a thermostat. Those will be removed in an upcoming release and should not be further extended.